tests/connections/GitlabRepoTest.ts

```diff
@@ -79,6 +80,17 @@ describe("GitLabRepoConnection", () => {
 				excludingLabels: ["but-not-me"],
 			} as GitLabRepoConnectionState as unknown as Record<string, unknown>);
 		});
+		it("will convert ignoredHooks for existing state", () => {
+			const state = GitLabRepoConnection.validateState({
+				instance: "foo",
```

Comment:
Add default correct state as const, then override.

Reply:
I think in the end this isn't quite noisy enough to warrant it, so I've left it for a future refactor.